### ISSUE TITLE
Wish: overrite Cell Values with inquiry

### ISSUE THREAD

**Issue by bastel42** [SecUSo/privacy-friendly-sudoku#68]
if a Cell allready contains a number and you slip on entering an value to wrong cell  this value is overritten.
Would it possible to implement  an option to ask for overwriting cells?

**Comment by annihat** [comment]
is this needed? The [back] button reverts the cell to previous values

**Comment by bastel42** [comment]
yes because, when playing quick you don't recognize that you have 
overriten a cell which allready have been set.


------ Originalnachricht ------
Von: "annihat" ***@***.***>
An: "SecUSo/privacy-friendly-sudoku" 
***@***.***>
Cc: "bastel42" ***@***.***>; "Author" ***@***.***>
Gesendet: 26.03.2021 11:19:28
Betreff: Re: [SecUSo/privacy-friendly-sudoku] overrite Cell Values with 
inquiry (#68)

**Comment by annihat** [comment]
> yes because, when playing quick you don't recognize that you have overriten a cell which allready have been set.
> […](#)

would a screen flash or a beep work instead? 
I ask because I think a pop-up would annoy me

**Comment by annihat** [comment]
can I up-vote for this enhancement, please?
